inference-engine/tests/functional/plugin/cpu/shared_tests_instances/behavior/core_integration.cpp

```diff
@@ -83,23 +83,23 @@ INSTANTIATE_TEST_CASE_P(
 
 INSTANTIATE_TEST_CASE_P(
         smoke_IEClassExecutableNetworkGetMetricTest, IEClassExecutableNetworkGetMetricTest_SUPPORTED_CONFIG_KEYS,
-        ::testing::Values("CPU", "MULTI:CPU", "HETERO:CPU", "AUTO"));
+        ::testing::Values("CPU", "MULTI:CPU", "HETERO:CPU", "AUTO:CPU"));
 
 INSTANTIATE_TEST_CASE_P(
         smoke_IEClassExecutableNetworkGetMetricTest, IEClassExecutableNetworkGetMetricTest_SUPPORTED_METRICS,
-        ::testing::Values("CPU", "MULTI:CPU", "HETERO:CPU", "AUTO"));
+        ::testing::Values("CPU", "MULTI:CPU", "HETERO:CPU", "AUTO:CPU"));
```

Comment:
I have concern that now we tests only `AUTO:CPU` or `AUTO:GPU` while not a combination of them. E.g. all logic for device selection selects among a single device

Reply:
I agree we need AUTO:CPU,GPU tests, since this may cause different behaviors, let me think and provide one test like AUTO:CPU,GPU. 
BTW, do you have any expected behavior for AUTO:CPU,GPU? I could add that to a test, too.

Reply:
And what do you think if I add tests for AUTO:CPU,CPU in `openvino/inference-engine/tests/functional/plugin/shared/src/behavior`? since I may not add too much tests for AUTO:CPU,GPU, I will create a test file and write test in this folder.

Reply:
And we just instantiate existing tests for GPU with one more `AUTO:CPU,GPU`?

Reply:
ok, maybe I could find a place to add one more `AUTO:CPU,GPU` case in existing tests. will try.

Reply:
oh, just got what you mean..., that's brilliant, I just add tests on core_integration.cpp, I may check others files to support such test cases.